File: JAVA/392. Is Subsequence.java
class Solution {
    public boolean isSubsequence(String s, String t) {
        int i,f=0;
        for(i=0;i<t.length() && f<s.length();i++){
            if(t.charAt(i)==s.charAt(f))
            f++;
        }
        return f==s.length();
    }
}

File: C++/392. Is Subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int f=0,i;
        for(i=0;i<t.size();i++){
            if(s[f]==t[i])
            f++;
        }
        return s.size()==f;
    }
};